lib/alephant/logger/levels_controller.rb

```diff
@@ -0,0 +1,42 @@
+module Alephant
+  module Logger
+    class LevelsController
+      LEVELS = %i(debug info warn error).freeze
+
+      class << self
+        def should_log?(message_level:, desired_level:)
```

Comment:
So if these are nil (unlikely as we control where this is called), this would end up resolving to 0 anyway. So we should be alright by not having defaults.

Reply:
Good shout, only desired_level will default to `0`; `message_level` will be nil (when not provided). Given the way the interface is set up, I think we should `return false` if either `message_level` or `desired_level` are nil. Thus, not loggable?

Reply:
In addition, we're raising an `ArgumentError` if the desired_level is an invalid type. I imagine we should only `return false unless message_level`. Since it's only the message level without a guard clause.